keps/sig-node/3619-supplemental-groups-policy/README.md

```diff
@@ -907,6 +969,8 @@ and creating new ones, as well as about cluster-level services (e.g. DNS):
       - Impact of its degraded performance or high-error rates on the feature:
 -->
 
+Specific version of CRI.
```

Comment:
@samuelkarp any comments on this? Is this a Containerd 2.0 - only feature?

Reply:
https://github.com/containerd/containerd/pull/9737 and https://github.com/containerd/containerd/pull/10410 will be in containerd 2.0.